pkg/descheduler/strategies/lownodeutilization_test.go

```diff
@@ -803,29 +803,21 @@ func TestWithTaints(t *testing.T) {
 				objs = append(objs, pod)
 			}
 
-			fakePtr := newFake(objs...)
```

Comment:
Is `newFake` called anywhere else? I don't see it used in this file anymore so we can remove it then right?